unit-tests/algo/projection/test-distortion.cpp

```diff
@@ -62,20 +54,11 @@ TEST_CASE( "brown_conrady_deproject" )
     compare(pixel1, pixel2);
 }
 
+#ifdef __SSSE3__
```

Comment:
Why SSE3? That's the only thing we work with?

Reply:
These tests are specifically for SSE.